### ISSUE TITLE
mouse mode change: bug or feature?

### ISSUE THREAD

**Issue by ruleh** [pelya/xserver-xsdl#41]
The mouse movement mode changes from non-relative mode (touchscreen mode) to relative mode (laptop mode) when rotating the screen to potrait and back to landscape. 
Sending the app to background and oppening it back up again makes the mouse behave like normal again (in this case touchscreen mode).

Now I am not sure of this is really a bug or just a very usefull feature.

If this is a bug, instead of removing it, could you turn it into some kind of feature where instead of rotating the screen to change the mouse mode, the user could simply press a button to do the changing?

**Comment by pelya** [comment]
Yes this is a bug. Are you using USB mouse or stylus? It should switch to
laptop mode when using finger, and to non-relative mode when using hardware
pointer, such as USB mouse.

**Comment by ruleh** [comment]
I was using the touchscreen the whole time.
However this bug also happens with a stylus.
(tested on galaxy s2 and galaxy note)
